test/Transforms/test-listener-canonicalize.mlir

```diff
@@ -0,0 +1,98 @@
+// RUN: mlir-proto-opt %s -allow-unregistered-dialect -test-listener-canonicalize --split-input-file | FileCheck %s
+
```

Comment:
Same thing here re copied from ?
Maybe just a top of the file comment that this is fully copied from core and to not modify?